api/ReadableStream.json

```
@@ -63,10 +63,10 @@
             "version_added": "30"
           },
           "safari": {
-            "version_added": null
+            "version_added": "9"
```

For Desktop, it was added in 10 https://developer.apple.com/documentation/webkitjs/readablestream
```suggestion
            "version_added": "10"
```

---

The linter doesn't allow top-level feature added later than sub features version, and it makes sense. There might be something off on webkit docs